app/src/app/services/content-management.service.ts

```diff
@@ -579,7 +579,7 @@ export class ContentManagementService {
         title: 'VERSION.DIALOG.TITLE'
       };
       this.newVersionUploaderService
-        .openUploadNewVersionDialog(newVersionUploaderDialogData, { width: '630px' })
+        .openUploadNewVersionDialog(newVersionUploaderDialogData, { width: '630px', role: 'dialog' })
```

Comment:
Is this role required? According to material documentation -> MatDialog creates modal dialogs that implements the ARIA role="dialog" pattern by default. You can check it out here: https://material.angular.io/components/dialog/overview#accessibility

Reply:
You are right MatDialog adds ARIA role by default. But, somehow it is not creating the mat-dialog-container with role="dialog".
As you can see the below screenshot
![image](https://user-images.githubusercontent.com/111955585/201077580-8ab72008-9676-4c7f-ad8e-b0e2445b940c.png)

Reply:
Have you tried investigating why this happens in our dialog? I mean why the role isn't applied as it should be

Reply:
I tried but did not get the reason of this.